package/Runtime/Scripts/ScriptableObjects/NimbusSDKConfiguration.cs

```diff
@@ -68,4 +71,12 @@ public class NimbusSDKConfiguration : ScriptableObject {
 			return str.All(c => c >= '0' && c <= '9');
 		}
 	}
+
+	public enum NimbusLogLevel
+	{
+		OFF,
+		ERROR,
+		DEBUG,
+		INFO
+	}
```

Comment:
Remove this this please. If enableUnityLogs == true set the log level automatically to the highest level. I'll be simplifying this scriptable object alot.